benchmarks/python/conftest.py

```diff
@@ -106,10 +110,30 @@ def pytest_configure(config):
         "markers",
         "inner_persistent: mark tests using inner_persistent scheduler if not being segmented.",
     )
+    config.addinivalue_line(
+        "markers",
+        "outer_persistent: mark tests using outer_persistent scheduler if not being segmented.",
+    )
+    config.addinivalue_line(
+        "markers",
+        "reduction: mark tests using reduction scheduler if not being segmented.",
+    )
+    config.addinivalue_line(
+        "markers",
+        "matmul: mark tests using matmul scheduler if not being segmented.",
+    )
     config.addinivalue_line(
         "markers",
         "resize: mark tests using resize scheduler if not being segmented.",
     )
+    config.addinivalue_line(
+        "markers",
+        "transpose: mark tests using transpose scheduler if not being segmented.",
+    )
+    config.addinivalue_line(
+        "markers",
+        "pointwise: mark tests using pointwise scheduler if not being segmented.",
```

Comment:
This comment indicates the marker is for fusion not segmented (due to limited register/smem size per SM), so we don't need to add `pointwise` marker to many normalization tests. Similar for `reduction` marker.